lighthouse-core/gather/gather-runner.js

```diff
@@ -29,8 +29,22 @@ const UIStrings = {
    */
   warningTimeout: 'The page loaded too slowly to finish within the time limit. ' +
   'Results may be incomplete.',
+  /**
+   * @description Warning that the host device where Lighthouse is running appears to have a slower
+   * CPU than the expected Lighthouse baseline.
+   */
+  warningSlowHostCpu: 'The device that ran this test appears to have a slower CPU than  ' +
+  'Lighthouse expects. This can negatively affect your performance score. Learn more about using ' +
```

Comment:
What action do we expect people to take if they see this warning?

The docs don't really spell it out.

They can buy a better machine.. or they can specify a `cpuSlowdownMultiplier`.. but... how would they calculate what number is good. Without guidance they can feel free to set `cpuSlowdownMultiplier` to 1 and enjoy some pretty great scores. 😛

Reply:
> What action do we expect people to take if they see this warning?

That they specify a lower cpuSlowdownMultiplier using the conversion table.

My expectation was that the below snippet from the linked docs was sufficient guidance on what to use, but it seems that was overly optimistic then :)

> Below is a table of the various cpuSlowdownMultipliers you might want to use to target different devices along with the possible range. If your device's BenchmarkIndex falls on the higher end of its bracket, use a higher multiplier from the range in the table below. If your device's BenchmarkIndex falls on the lower end of its bracket, use a lower multiplier from the range in the table below. If it's somewhere in the middle, use the suggested multiplier.

Suggestions on alternative wording that's more direct?

Reply:
Bump on what you would like to see in the docs beyond the quoted passage :)

Reply:
> Bump on what you would like to see in the docs beyond the quoted passage :)

A table with recommended `cpuSlowdownMultiplier` depending on the benchmark_index I get.

I have several instances of lighthouse running in different devices with benchmarkIndex between 550 - 800 with a `cpuSlowdownMultiplier` of 4x. According to this, my instances are severely underpowered. What should I do? Change the `cpuSlowdownMultiplier` to 2x? 1.5x?

Reply:
@dvelasquez thanks for the extra info! Were all of these values using Chrome Canary (or at least m86 or later)? All of these values are for Chrome 86 or later (which changed the scale a bit) and this change would be accompanying Chrome 87.

Maybe I'll just make a straight up calculator UI for this then if that's what folks are looking for.

Reply:
@patrickhulce still on Chrome 85 and LH 6.3.0. Since I have some logic in place that drops the runs below 500, I want to be sure when do the swap and which values use as a threshold (I think it should be 1000 now, right?)

Reply:
@paulirish how's https://lighthouse-cpu-throttling-calculator.vercel.app/?

Reply:
we could link with a query param for the BenchmarkIndex from the report to prepopulate the command?

Reply:
Amazing!

Reply:
> I know this is a far shot but could be possible to apply the multiplier after the gatherers? So apply the correct cpuSlowdownMultiplier after we know the benchmarkIndex score?

It is possible and we explored doing that in an earlier PR, but the variance of benchmark index and inconsistency across machines (that's why the calculator gives you a *range*, not just a single number) means this would introduce further unwanted variance, so we abandoned it. #9085 contains the full story.

Reply:
Is there a way to get the benchmarkIndex without run a full lighthouse analysis?
I'm looking for something to "probe" the machines every once in a while to check the multipliers are in the range.

Reply:
Yes, you can import the function directly from node and run that 

https://github.com/GoogleChrome/lighthouse/blob/054722a5c895646abefb777ab6744549bbd9eade/lighthouse-core/lib/page-functions.js#L472